Hippo.Core/Domain/QueuedEvent.cs

```diff
@@ -0,0 +1,64 @@
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
+namespace Hippo.Core.Domain;
+
+public class QueuedEvent
+{
+    public int Id { get; set; }
+    [MaxLength(50)]
+    public string Action { get; set; } = "";
+    [MaxLength(50)]
+    public string Status { get; set; } = "";
+    public string Data { get; set; } = "";
+    public string ErrorMessage { get; set; } = "";
+    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
+    public DateTime UpdatedAt { get; set; } = DateTime.UtcNow;
+    public int? RequestId { get; set; }
```

Comment:
how can you have a queued event that doesn't have an attached request?

Reply:
Requests are for actions that need to be approved. There's no request associated with a user updating their ssh key.